bundles-lib/pom.xml

```diff
@@ -0,0 +1,179 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
```

Comment:
bundles-lib started from nifi-nar-utils, but then I made the refactoring changes spelt out above.

I could add to the README.md if it is insufficient.